packages/asset-swapper/package.json

```diff
@@ -11,7 +11,7 @@
         "build": "yarn tsc -b",
         "watch": "tsc -w -p tsconfig.json",
         "build:ci": "yarn build",
-        "lint": "tslint --format stylish --project .",
+        "lint": "tslint --format stylish --project . && prettier --check 'src/**/*.{ts,tsx,json,md}' --config ../../.prettierrc",
```

Comment:
We have the established pattern of having `prettier` as it's own script: https://github.com/0xProject/0x-monorepo/blob/development/packages/orderbook/package.json#L25

Reply:
I introduced a `prettier` script in https://github.com/0xProject/0x-monorepo/pull/2541/commits/eb5ec58453bca786806c25dfe1b9f4269fca4ed4, but I also kept its invocation in the `lint` script.

I feel pretty strongly that `yarn lint` should run everything that CI's `static-tests` runs.  I'm open to having my mind changed if anyone else feels strongly about it, but that's why I did it this way.

Reply:
I think it's pretty weird to have different `lint` behavior for `asset-swapper` compared to all other packages.   IMHO it's important to be consistent here.

Unless we want to make this change wholesale across all repos (a can of worms I don't think we should open right now), I think we should make this consistent.